src/ui/preview/Preview.vue

```diff
@@ -91,7 +107,13 @@ export default {
         }
     },
     destroyed() {
-        this.view.destroy();
+        if (!this.existingView) {
+            this.view.destroy();
+        } else if (this.existingViewElement) {
+            // if the existing view element is populated, it's the currently viewed view
+            // in preview and we need to add it back to the parent.
+            this.addExistingViewBackToParent();
```

Comment:
wondering if we can do this in `src/plugins/viewLargeAction/viewLargeAction.js`? passed down element append back to parent?

Reply:
I would, but the way this is set up now, we're storing the view back on the parent when they select another view (ie. telemetry table), this way the view is still active somewhere and if they close from there, it's just back where it belongs anyways.